src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -258,6 +259,15 @@ export class ExposureNotificationService {
   async updateExposureStatusInBackground() {
     // @todo: maybe remove this gets called in updateExposureStatus
     if (!(await this.shouldPerformExposureCheck())) return;
+
+    const filteredMetricsService = FilteredMetricsService.sharedInstance();
+
+    const taskStartDate = getCurrentDate();
+    const taskIdentifier = Math.floor(Math.random() * 100) + 1;
```

Comment:
Since the goal is just to differentiate a set of events between two background tasks occurence I think we should be fine with a random number between 0 and 100

Reply:
Could this just be an incremental counter? We have access to the previous records n'est pas?

Reply:
Could this just be an incremental counter? We have access to the previous records n'est pas?

Reply:
There is no API to get a specific stored event so it would add more complexity to the code but we could definitely find a simpler solution if we really want to push that to production